docker_custodian/docker_gc.py

```diff
@@ -38,8 +50,43 @@ def cleanup_containers(client, max_container_age, dry_run):
             container['State']['FinishedAt']))
 
         if not dry_run:
-            api_call(client.remove_container, container=container['Id'],
-                     v=True)
+            api_call(
+                client.remove_container,
+                container=container['Id'],
+                v=True,
+            )
+
+
+def filter_excluded_containers(containers, exclude_container_labels):
+    def include_container(container):
```

Comment:
What about preprocessing `exclude_container_labels` before passing it down to `should_exclude_container_with_labels()`?

Also, another optimization is to return the compete `containers` upon empty `exclude_container_labels`.

Reply:
Sure, passing down formatted exclude labels rather than the raw args
Also added the short circuit optimization on empty `exclude_container_labels`